mantis-common/src/main/java/io/mantisrx/runtime/descriptor/StageScalingPolicy.java

```diff
@@ -162,7 +144,8 @@ public enum ScalingReason {
         ClutchRps,
         RPS,
         JVMMemory,
-        SourceJobDrop
+        SourceJobDrop,
+        FailoverAware
```

Comment:
what does ScalingReason::FailoverAware mean? I thought these were sent as events from the metric handler to the autoscaling policy.

Reply:
these are indeed sent as events but then we also need a failover autoscaling strategy similar to a `CPU` or `UserDefined` autoscaling strategy.